### ISSUE TITLE
Error when trying to import worlds and improper loading of imported worlds.

### ISSUE THREAD

**Issue by InuYasha86000** [Multiverse/Multiverse-Core#1625]
I cannot seem to import any of my worlds into multiverse 1.7.10-2.5, i'm creating my own mod pack server running KCauldron-1.7.10-1448.133-server and have tried importing my worlds from an old 1.7.10 backup and from a recent 1.8 version and neither will import properly. I either get "Error , Cannot Import World" or something close to that, or it imports them but returns them to a fresh untouched world. other than the Error about not being able to import a world I get no other errors.  any ideas what's going on?

**Comment by InuYasha86000** [comment]
I have updated the server to Kcauldron 1566 and the same issue is still present. it won't even recognize a previous world generated using the server itself.

**Comment by TWSSYesterday** [comment]
Update to latest KCauldron and try http://ci.onarandombox.com/job/Multiverse-Core/694/artifact/target/Multiverse-Core-2.5.jar Pastebin startup logs

**Comment by InuYasha86000** [comment]
I will have to use my dropbox for one of the logs as it's so big but here's the pastebin of the latest client log including an import attempt:http://pastebin.com/0tr2W2vS
and here's the dropbox link to the server side log: https://www.dropbox.com/s/zwcmyp9ntmv16s2/latest%20Kcauldron%20log.log?dl=0 once again the 2nd log is too big for pastebin that is why I dropboxed it.

**Comment by InuYasha86000** [comment]
is there anything else I can try and supply, and is this able to be replicated by you?

**Comment by InuYasha86000** [comment]
I have found something else out. for some reason multiverse is looking for world files within the main world folder. I tried to place a world within the main world folder and import it, the import DID work, however it then booted me from the server and when I logged back in it had replaced a bunch of modded items I had built with completely different items, pretty much ruining the world. but the imported world seems to be ok, no irregularities found so far.
